frontend/src/app/components/Routes.js

```diff
@@ -1,36 +0,0 @@
-import React from 'react';
```

Comment:
We don't need the Routes file any more, since now the routes are just specified by .js and .md files in the frontend/src/pages directory.